DigitalLearningSolutions.Data.Tests/Helpers/CustomPromptsTestHelper.cs

```diff
@@ -0,0 +1,100 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Helpers
+{
+    using DigitalLearningSolutions.Data.Models.CustomPrompts;
+
+    public static class CustomPromptsTestHelper
+    {
+        public static CentreCustomPrompts GetDefaultCentreCustomPrompts
+        (
+            int centreId = 2,
```

Comment:
Could we use centre 29 for the custom prompts data service tests as it has two custom prompts (one with options, one without) - this means that we can actually test some of these joins working correctly.